### DISCUSSION TITLE
Bookmarks file location

### DISCUSSION THREAD

**Discussion by RafaelLinux** [mate-desktop/caja#1788] [Q&A] [answered]
Hello
Please, can someone tell me where is located Caja bookmarks file to make backups?

Thank you

**Comment by lukefromdc** [comment]
I have never worked on the bookmarks code, so I don't know where they are stored but the file with much of the code for making bookmarks is libcaja-private/bookmarks.c

**Comment by RafaelLinux** [comment]
Thank you for the information. I'm not a programmer, I'm a user, so I doubt I'll be able to find out from the code where this information is stored. Is there no one who has a general knowledge about the files Caja uses?

**Comment by lukefromdc** [comment]
A lot of folks seem to be taking a break now and I cannot handle the whole project myself

**Comment by lukefromdc** [comment]
Also note that we don't have any paid/fulltime people at all

**Comment by RafaelLinux** [comment]
Don't worry, I'll try to find out somehow. I figured it's an unpaid project and I didn't mean to bother, I just thought that there was a documentation about it, because from the application itself there is no option to ‘import’ or ‘export’ the bookmarks, which would be very useful for this case, but since it doesn't exist, that's why I was asking. I imagined that they were saved in a file in a hidden folder in $HOME but I don't think it's that simple.

Thanks anyway, the application works better than other better known ones.

**Comment by lukefromdc** [comment]
No problem. Such documentstion might be somewhere, I just haven't seen it.

**Comment by fracpete** [comment, marked as answer]
You can find the bookmarks here:
```
$HOME/.config/gtk-3.0/bookmarks
```

**Comment by RafaelLinux** [comment]
You really know! That's exactly where it is. I never imagined it could be under the GTK subfolder, so I had given up on it. Your contribution really made my day.

Thank you!!
